Code/src/main/java/nl/utwente/group10/ui/components/lines/ConnectionLine.java

```diff
@@ -46,8 +48,11 @@ public ConnectionLine() {
 
     /**
      * Sets the start position for this Line object
-     * @param x coordinate
-     * @param y coordinate
+     * 
+     * @param x
+     *            coordinate
+     * @param y
+     *            coordinate
```

Comment:
While I believe the modified version is correct to present correctly formatted JavaDoc I changed the doc a little to make it fit on one line.